### DISCUSSION TITLE
What changes and behaviors are planned for other mobs?

### DISCUSSION THREAD

**Discussion by Fandovec03** [VirusTLNR/LethalIntelligence#10] [General]
Hi! I am currently looking into creating a mod that would make mobs and the fauna feel more alive. Currently my mod makes mobs (rn only spider) hit enemy when they collide. I plan to eventually create an ecosystem and unique situation whenever 2 mobs encouner.

I am interested in eventually adding a compatibility with mods altering Masked behavior, especially this one.

**Comment by VirusTLNR** [comment]
Hi,

Your ideas and mine may end up conflicting/being incompatible, but I'm happy to discuss it.

I am kind of on the fence about making different monsters interact with each other, on the fence meaning, thought about adding it myself eventually (which is why it may conflict with yours), and also paused for thought on the ramifications of it all. So yeah happy to hear your ideas and thoughts on it all as another mod doing it is a nice solution which means if I don't want to do it, I don't need to touch stuff like that, but van integrate with yours for people who want it.

**Reply by Fandovec03** [reply]
Gotcha. Rn my mod rn is really only good for slime as I can define which enemies to damage and which to ignore but in the future I'd like to add behaviors such as hunting other entities. In case of masked your mod pretty much accomplishes my goal: Make the world feel more alive.

It's gonna take a while to materialize a product worth releasing though and I still have a lot to learn about modding lethal.
